src/main/java/net/javadiscord/javabot/systems/commands/FormatCodeCommand.java

```diff
@@ -28,6 +28,9 @@ public ReplyCallbackAction handleSlashCommandInteraction(SlashCommandInteraction
 		if (idOption == null) {
 			if (event.getChannel().hasLatestMessage()) {
 				id = event.getChannel().getLatestMessageIdLong();
+				if (event.getChannel().retrieveMessageById(id).complete().getAuthor().isBot()){
+					return Responses.error(event, "Missing required arguments.");
+				}
```

Comment:
Don't use `.complete()` calls, as they're blocking the current thread. In addition, I think it would be useful if it takes the last message that isn't sent from a bot instead of throwing an error.

Reply:
fetch last 5 messages and format latest non bot message?

Reply:
for example, yes